spec/rails_helper.rb

```diff
@@ -69,6 +69,7 @@
   end
 
   config.around(:each) do |example|
+    Settings.reload!
```

Comment:
Oh, I see - I think this answers my question above. How much time does this take?